### DISCUSSION TITLE
Vote: DeviceIpv4Addr object properties (from PR #139)

### DISCUSSION THREAD

**Discussion by hdamker** [camaraproject/QualityOnDemand#141] [Polls]
Dear QoD team,

Please provide your vote on the question below.

Option 1 is the proposal currently used within the PR #139 by @eric-murray 
Option 2 is the proposal presented within [UE.IP.address.and.ports.in.QoD.PA3.pdf](https://github.com/camaraproject/QualityOnDemand/files/11013180/UE.IP.address.and.ports.in.QoD.PA3.pdf) by @emil-cheung 

For further background see the discussion in issue #34 and PR #139. 

In the interests of transparency, and to attempt to maintain the one company one vote policy, I'd suggest that anyone who votes puts their vote in the comments.

Thanks,
Herbert

**Comment by hdamker** [comment]
Deutsche Telekom votes for Option 1.

**Comment by ShutingQing** [comment]
Feel like public private is more likely a common language. I think a comment or note needs to be applied to explain what is observed addr. Personal idea.

**Reply by ShutingQing** [reply]
voted 1

**Comment by patrice-conil** [comment]
Orange votes for option 1

**Comment by konomith** [comment]
TELUS votes for option 1

**Comment by jlurien** [comment]
TEF votes for option 1

**Comment by hdamker** [comment]
Decision to take option 1.
